assets/src/edit-story/components/canvas/carousel/gridView.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Save all svg in another directory to keep it clean.

Reply:
Makes sense. Some icons like the arrows might be shared with other components, so we can probably have a central directory for those ones.

Reply:
@swissspidy Note that in another PR I "stole" your arrows and moved these to the buttons directory, also created buttons for these which could be used anywhere:
https://github.com/ampproject/amp-wp/pull/4049/files#diff-afde5e2694872f1beb0fc59de477fb11R11-R12

Reply:
Sweet! Thanks @miina

Reply:
Changed in c68e8bf02f7dddd39166fcfcd1684c7f6b9e937f